misc/dbt-materialize/dbt/include/materialize/macros/columns.sql

```diff
@@ -22,3 +22,28 @@
         {{ select_sql }}
     ) as __dbt_sbq
 {% endmacro %}
+
+{% macro materialize__get_empty_schema_sql(columns) %}
+    {%- set col_err = [] -%}
+    {%- set col_naked_numeric = [] -%}
+    select
+    {% for i in columns %}
+      {%- set col = columns[i] -%}
+      {%- if col['data_type'] is not defined -%}
+        {%- do col_err.append(col['name']) -%}
+      {#-- If this column's type is just 'numeric' then it is missing precision/scale, raise a warning --#}
+      {%- elif col['data_type'].strip().lower() in ('numeric', 'decimal', 'number') -%}
+        {%- do col_naked_numeric.append(col['name']) -%}
+      {%- endif -%}
+      {% set col_name = adapter.quote(col['name']) if col.get('quote') else col['name'] %}
+      -- NOTE(morsapaes): in a future release of dbt, the default macro will use
+      -- the global cast macro, as modified here, and we can remove this custom
+      -- override. See: https://github.com/dbt-labs/dbt-adapters/pull/165
```

Comment:
Oh, this was already released in [`dbt-adapters` v1.1.0](https://github.com/dbt-labs/dbt-adapters/releases/tag/v1.1.0). Still getting used to the new package structure. 🫠